.github/ISSUE_TEMPLATE/bugreport.yml

```diff
@@ -0,0 +1,39 @@
+name: Issue form
+description: Create an issue for the CMake Tools extension.
+body:
+- type: markdown
+  attributes:
+    value: |
+      ### Is there an existing issue for this?
+      Please search our [existing issues](https://github.com/microsoft/vscode-cmake-tools/issues) to see if an issue already exists for the issue you'd like to report.
+- type: textarea
+  attributes:
+    label: Brief Issue Summary
+    description: |
+      Put a short summary here.
+      If this is a bug, please provide clear instructions as to how we can reproduce the issue on our end.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: CMake Tools Diagnostics
+    description: |
+      For bugs, please run the `CMake: Log Diagnostics` command and paste the output here.
```

Comment:
Is it possible to tell the form to format this like code \`\`\`?

Reply:
`render: shell` maybe?

Reply:
Made the input for logs to render as shell.